manage.py

```diff
@@ -131,6 +131,12 @@ def patch_pickle():
         import pickle5
         sys.modules["pickle"] = pickle5
 
+        # HACK set HIGHEST_PROTOCOL to one understood by Python 3.6.
+        # Celery uses HIGHEST_PROTOCOL to make pickles and Kombu, a
+        # dependency of Celery, uses the built-in pickle module
+        # (imported before we have a chance to patch it).
+        pickle5.HIGHEST_PROTOCOL = pickle5.DEFAULT_PROTOCOL
```

Comment:
After reviewing how `HIGHEST_PROTOCOL` is used internally in `pickle5`, I think this is not a good solution. I think this will make it harder to read protocol 5 pickles, which is the whole point of using pickle5.

Reply:
Actually, I take that back. As long as the optimized C version of pickle is available it will use that, which does not use the constant. I think this will work.